### PR TITLE
Remove blank entities. Conditionally load mapping if FOSUser present.

### PR DESCRIPTION
**Pull request by rn0**

### CODE REVIEW

**Comment by rn0** [comment]
@fsi-open

File: Resources/config/doctrine/legacy/FOSUser.orm.xml [hunk 1]
```diff
@@ -4,11 +4,11 @@
                   xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping
                   http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
 
-    <mapped-superclass name="FSi\Bundle\AdminSecurityBundle\Entity\FOSUser">
+    <mapped-superclass name="FSi\Bundle\AdminSecurityBundle\Security\FOSUser\FOSUser">
```
Comment (szymach) [inline Resources/config/doctrine/legacy/FOSUser.orm.xml]: FOSUser\FOSUser? maybe FOS\User?

File: FSiAdminSecurityBundle.php [hunk 2]
```diff
@@ -21,6 +22,22 @@ public function build(ContainerBuilder $container)
...
+
+        $doctrineConfigDir = realpath(__DIR__ . '/Resources/config/doctrine');
+
+        $mappings = array(
+            $doctrineConfigDir . '/model' => 'FSi\Bundle\AdminSecurityBundle\Security\Model',
+        );
+
+        $container->addCompilerPass(DoctrineOrmMappingsPass::createXmlMappingDriver($mappings));
+
+        if (class_exists('FOS\UserBundle\Entity\User')) {
```
Comment (JarekW) [inline FSiAdminSecurityBundle.php]: what if fos-user is in vendors, but it is not enabled bundle? I don't really know if this is case we need to care here

**Comment by JarekW** [comment]
:+1:

File: FSiAdminSecurityBundle.php (hunk 2 above)
Comment (chives) [inline FSiAdminSecurityBundle.php]: you should use ``$container->hasExtension()`` method

**Comment by rn0** [comment]
@fsi-open

**Comment by rn0** [comment]
@fsi-open

**Comment by szymach** [comment]
@fsi-open

**Comment by chives** [comment]
:+1:

**Comment by JarekW** [comment]
:+1:

**Pull request merged**